services/workflows-service/src/auth/assignee-asigned-guard.service.ts

```diff
@@ -0,0 +1,19 @@
+import { CanActivate, ExecutionContext, Injectable } from '@nestjs/common';
+import { Request } from 'express';
+import { WorkflowService } from '@/workflow/workflow.service';
+import { WorkflowDefinitionUpdateInput } from '@/workflow/dtos/workflow-definition-update-input';
+import { WorkflowRuntimeData } from '@prisma/client';
+import console from 'console';
+
+@Injectable()
+export class AssigneeAsignedGuard implements CanActivate {
```

Comment:
This guard relays on workflow data, the name should probably suggest it

`WorkflowAssigneeGuard`